lib/rpush/daemon/dispatcher/http.rb

```diff
@@ -6,6 +6,7 @@ def initialize(app, delivery_class, _options = {})
           @app = app
           @delivery_class = delivery_class
           @http = Net::HTTP::Persistent.new(name: 'rpush')
+          @http.proxy = Rpush.config.proxy if Rpush.config.proxy.present?
```

Comment:
This only applies to HTTP dispatchers, and does not contemplate APNSv2/p8. Could you maybe add it to those dispatchers also?

Reply:
Sorry but I didn't use APNS, so I cannot test it

Reply:
Would a spec not be enough to create confidence in its functionality?